templates/rabbitmq.config.erb

```diff
@@ -1,6 +1,9 @@
 % This file managed by Puppet
 % Template Path: <%= @module_name %>/templates/rabbitmq.config
 [
+<% if @ssl -%>
+  {ssl, [{versions, [<%= @ssl_versions.sort.map { |v| "'#{v}'" }.join(', ') %>]}]},
```

Comment:
There is already an `if @ssl` section on line 21, this should be within that.

As mentioned in my other comment I think `@ssl_versions` should default to `undef` and have this line wrapped in `if @ssl_versions`. I'm seeking a second opinion now.